_adr/89/index.adoc

```diff
@@ -0,0 +1,116 @@
+Renewals will be blast radius aware to renew certificates in small runs.
+The intention is to reduce the risk of compromising all our certificates in case there is an outage or an issue with the renewal system in Letsencrypt. 
+It also removes the chances of bombarding Letsencrypt with bulk requests or rollout of all the Kafkas in one big batch after renewals. 
+5. Revocation will be handled per data plane cluster and it will follow a GitOps approach where SRE will request to revoke a certificate for a given data plane.
+Once the compromised certificate has been revoked, a new certificate for that data plane will be generated.
+For already existing data planes they'll continue to share the newly generated certificate: see proposal number (3). 
+6. When a data plane is removed from the fleet manager, its corresponding certificate will also be revoked. 
+If the data plane in question already existed before the solution is in place ( i.e it shares a certificate with other data plane as described in proposal number (3)), then revocation of the certifcate will only happen if there is no remaining data plane that uses it. 
+
+NOTE: There is an existing logic in the data plane that ensures that all sensitive parts in the ManagedKafka CR are not written in plain text. 
+Instead a secret is created and only references to the secrets will be added before the ManagedKafka is created in the data plane.
+
+### Threat model
+// Provide a link to the relevant threat model. 
+// You must either update an existing threat model(s) to cover the changes made by this ADR, or add a new threat model.
+
+Once the certificate are in the data plane in form of a Kubernetes secret, a priviledged users can still see them. 
+If this is on a enterprise data plane cluster without good security standards/policy, then anyone with the private key can impersonate the server or eavesdrop on a connection with the server. 
+For non enterprise data plane clusters, this is mitigated by only allowing SREs to have the priviledge / permissions to see these secrets.
+For enterprise data plane clusters, customers will be advised to do so as well.
+
+Morever, since there will be a different certificate per data plane clusters, if one certificate is compromised, it won't affect others and it can be safely revoked. 
+The proposed solution also limits the possibility of an owner of a enterprise data plane cluster, eavesdropping on another data plane cluster that they do not own as the certificate keys will be different.
+
+
+## Alternatives Considered / Rejected
+
+1. Deploying https://www.redhat.com/sysadmin/cert-manager-operator-openshift[cert-manager operator] to the data plane. Rejected to avoid exposing domain registrar credentials and to avoid adding another component that consumes data plane resources and that needs to be monitored.
+
+2. Migrating existing data plane (and their Kafkas). This change is disruptive to existing customers hence why it was rejected. 
```

Comment:
I understand but then let's put in the threat model that this  wildcard certificate of a lower level will allow impersonation of the enterprise kafka cluster.